prime-router/build.gradle.kts

```diff
@@ -434,6 +434,10 @@ dependencies {
     implementation("org.commonmark:commonmark:0.17.2")
     implementation("com.google.guava:guava:30.1.1-jre")
     implementation("com.helger.as2:as2-lib:4.7.1")
+    // Prevent mixed versions of these libs based on different versions being included by different packages
+    implementation("org.bouncycastle:bcpkix-jdk15on:1.69")
+    implementation("org.bouncycastle:bcmail-jdk15on:1.69")
+    implementation("org.bouncycastle:bcprov-jdk15on:1.69")
 
```

Comment:
Bouncy Castle did a release that caused mixed versions of their packages to be used. This change causes us to use the latest version for all BC packages.